eng/Versions.props

```diff
@@ -62,11 +62,11 @@
     <_MicrosoftWebWebView2Version>1.0.2088.41</_MicrosoftWebWebView2Version>
     <!-- GLIDE - the android maven artifact in /src/Core/AndroidNative/maui/build.gradle -->
     <!-- must be kept in sync with the binding library version to it here: -->
-    <_XamarinAndroidGlideVersion>4.15.1.2</_XamarinAndroidGlideVersion>
+    <_XamarinAndroidGlideVersion>4.16.0.1</_XamarinAndroidGlideVersion>
```

Comment:
When updating this, the Java side needs to match:
https://github.com/dotnet/maui/blob/baa86b20786f75632dd99ae13df7d268a5b843f9/src/Core/AndroidNative/maui/build.gradle#L29